lib/gds_api/test_helpers/account_api.rb

```diff
@@ -95,6 +95,18 @@ def stub_account_api_set_attributes(govuk_account_session: nil, attributes: nil,
             .to_return(status: 200, body: { govuk_account_session: new_govuk_account_session }.compact.to_json)
         end
       end
+
+      def stub_account_api_get_attributes_names(govuk_account_session: nil, attributes: [], new_govuk_account_session: nil)
+        querystring = Rack::Utils.build_nested_query({ attributes: attributes }.compact)
+        if govuk_account_session
```

Comment:
How doe sthe the else block work here?
It seems to return the same thing without headers.

Wouldn't this be an authenticated endpoint that returns 401 if there's not session?